docs/4.1. Behaviour - RTP Transport Type.md

```diff
@@ -64,6 +64,12 @@ Senders and Receiver Connection Management endpoints must support a core paramet
 ## Interpretation of SDP Files
 All Senders and Receivers should comply with RFC 4566 (Session Description Protocol) when creating and parsing SDP files. Multicast Senders and Receivers should additionally comply with RFC 4570 (SDP Source Filters). Two examples are given below, showing an SDP file and the transport parameters that would be presented at the `/staged` endpoint by a Receiver that has parsed the file.
 
+In the event that a receiver is presented with an SDP file and a set of transport parameters that
+contradict each other in the same PATCH request, the transport parameters take priority
+over the SDP file and must be the parameters used for activation. This only applies
+in the case where there is contradiction between the SDP file and transport parameters within
+a single PATCH request, in all other cases the latest PATCH request takes priority.
```

Comment:
Need to clarify "latest"?  Based on the order in which the the PATCH requests are received?